translations/de.js

```diff
@@ -1,102 +1,121 @@
   "Activate the global connect tool": "Globales Verbindungswerkzeug aktivieren",
   "Activate the hand tool": "Hand-Tool aktivieren",
   "Activate the lasso tool": "Lasso-Tool aktivieren",
   "Ad-hoc": "Ad-hoc",
   "Add Lane above": "Lane oberhalb hinzufügen",
   "Add Lane below": "Lane unterhalb hinzufügen",
+  "Append ConditionIntermediateCatchEvent": "ConditionIntermediateCatchEvent anhängen",
+  "Append EndEvent": "Endereignis anhängen",
+  "Append Gateway": "Gateway anhängen",
+  "Append Intermediate\/Boundary Event": "Angeheftetes / Zwischenereignis anhängen",
+  "Append MessageIntermediateCatchEvent": "Eintretendes Nachrichten-Zwischenereignis anhängen",
+  "Append ReceiveTask": "Empfangen Aufgabe anhängen",
+  "Append SignalIntermediateCatchEvent": "Eintretendes Signal-Zwischenereignis anhängen",
+  "Append Task": "Aufgabe anhängen",
+  "Append TimerIntermediateCatchEvent": "Eintretendes Zeit-Zwischenereignis anhängen",
   "Append compensation activity": "Kompensation hinzufügen",
   "Append {type}": "{type} anfügen",
   "Business Rule Task": "Geschäftsregel Aufgabe",
   "Call Activity": "Aufruf-Aktivität",
   "Cancel Boundary Event": "Angeheftetes Abbruch-Zwischenereignis",
-  "Cancel End Event": "Abbruchs-Endereignis",
   "Change type": "Typ ändern",
-  "Collapsed Pool": "Zugeklappter Pool",
+  "Collection": "Collection",
   "Compensation Boundary Event": "Angeheftetes Kompensations-Zwischenereignis",
   "Compensation End Event": "Kompensations-Endereignis",
   "Compensation Intermediate Throw Event": "Ausgelöstes Kompensations-Zwischenereignis",
   "Compensation Start Event": "Kompensations-Startereignis",
   "Complex Gateway": "Komplexes Gateway",
-  "Conditional Boundary Event (non-interrupting)": "Angeheftetes bedingtes Zwischenereignis (nicht unterbrechend)",
   "Conditional Boundary Event": "Angeheftetes bedingtes Zwischenereignis",
+  "Conditional Boundary Event (non-interrupting)": "Angeheftetes bedingtes Zwischenereignis (nicht unterbrechend)",
+  "Conditional Flow": "Bedingter Sequenzfluss",
   "Conditional Intermediate Catch Event": "Eintretendes bedingtes Zwischenereignis",
-  "Conditional Start Event (non-interrupting)": "Bedingtes Startereignis (nicht unterbrechend)",
   "Conditional Start Event": "Bedingtes Startereignis",
+  "Conditional Start Event (non-interrupting)": "Bedingtes Startereignis (nicht unterbrechend)",
   "Connect using Association": "Mit Assoziation verbinden",
   "Connect using DataInputAssociation": "Mit DataInputAssociation verbinden",
-  "Connect using Sequence/MessageFlow or Association": "Mit Sequenzfluss/Nachrichtenfluss oder Assoziation verbinden",
-  "Create IntermediateThrowEvent/BoundaryEvent": "IntermediateThrowEvent/BoundaryEvent erzeugen",
-  "Create Pool/Participant": "Pool/Teilnehmer erzeugen",
+  "Connect using Sequence\/MessageFlow or Association": "Mit Sequenzfluss/Nachrichtenfluss oder Assoziation verbinden",
```

Comment:
It seems like you mixed up the keys. What is `/` in the original key is `\/` in your updated translation.

Reply:
This comes in via the localazy export. It seems to escape also in keys. I fixed this, but eventually it would make sens to me to use better keys, without blanks, special chars, etc.
Also seems like there are slight differences in the sorting algorithm applied here and in localazy's export.